scss/_variables.scss

```diff
@@ -492,26 +492,26 @@ $table-cell-padding-x-sm:     .25rem !default;
 
 $table-cell-vertical-align:   top !default;
 
-$table-color:                 $body-color !default;
-$table-bg:                    transparent !default;
+$table-color:                 var(--bs-table-color) !default;
+$table-bg:                    var(--bs-table-bg) !default;
 
 $table-th-font-weight:        null !default;
```

Comment:
This can be set a CSS variable on application side by users, so that they can create own variants.